body.tex

```diff
@@ -0,0 +1,481 @@
+Currently, the letter $v$ is used.
+
+
+\subsection{Other Related Artifacts}
+
+Together with the creation of the release candidate, other artifacts need to be branched:
+
+\begin{itemize}
+\item Documentation: to consolidate all information relevant for the new release. This includes the release note.
+As an example, for Science Pipelines, the package \textit{pipelines\_lsst\_io} need to be branched.
+\item Environment definition packages, like for example packages containing conda environment files (YAML).
+\item Distribution packages; still referring to the Science Pipelines, the package \textbf{lsst}  where \textit{newinstal.sh} is developed, need to be branched also.
+\item Test data packages.
+\item Other packages not tagged automatically, but required to be in line with the released software.
+\end{itemize}
+
+
+\subsection{Release Candidates Validation} \label{sec:rcvalidation}
+
+The release candidate needs to be validated.
+
+In an ideal case, a test campaign following a specific test plan should be conducted.
+This test campaign should demonstrate that the release candidate, and therefore the forthcoming release, is behaving as required and expected.
+T/CAM and product owner shall be responsible for this activity.
+
+Practically, the validation in many cases can be:
+
+\begin{itemize}
+\item installation/configuration of the binaries packages, or distribution image; usually done manually
+\item inspect of the installation, that all expected files and configuration are there
+\item execution of some demo package available case by case depending on the SW product
+\item try some use cases in order to prove that the release candidate is behaving properly
+\item ask downstream users to act as beta testers.
+\end{itemize}
+
+For Science Pipelines, a characterization report is produced by the product owner, in order to document the release outputs.
+This is a new document created for each major release.
+
+
+\subsection{Resolving Problems}
+
+In case problems are found during the validation, a DM issue needs to be created in Jira.
+This issue shall follow the usual development process as described in the developer guide.
+The fix will be first implemented in a ticket branch, reviewed and merged to master.
+Once the fix has been proved to work, it can be backported to the release branch.
+
+The release branch will be created only for the packages to be fixed and will follow the naming convention described in \ref{sec:branchnaming}.
+
+The developer guide will provide examples of backporting procedures (\jira{DM-19950}).
+
+In a special case, that an issue cannot be fixed on master, the ticket branch can be opened based on the release branch.
+
+The porting can be applied also in case that an issue, implemented on master after the first release candidate has been created, has to be included in the release.
+
+The DMCCB has to overview the backporting process and take corrective actions if needed. 
```

Comment:
On HSC, we need a weekly call between the testers in NAOJ and the dev team in Princeton to identify and discuss the importance of specific issues for backporting

Reply:
T/CAMs shall be involved by the DMCCB.